images/php/7.1/Dockerfile

```diff
@@ -44,6 +44,22 @@ RUN docker-php-ext-install \
   xsl \
   zip
 
+
```

Comment:
Let's remove this extra blank line, keep just the `RUN` here and remove all of the other updates

Reply:
I have made the requested changes.  Thank you!